app/delivery/send_to_providers.py

```diff
@@ -129,7 +129,11 @@ def send_email_to_provider(notification):
                     buffer = response.read()
                     mime_type = magic.from_buffer(buffer, mime=True)
                     if mime_type == 'application/pdf':
```

Comment:
This logic will need to be updated when you'll work on choosing if services want to attach OR link to documents

Reply:
thanks